mteb/tasks/BitextMining/multilingual/NTREXBitextMining.py

```diff
@@ -75,8 +181,8 @@ class NTREXBitextMining(AbsTaskBitextMining, CrosslingualTask):
         annotations_creators="expert-annotated",
```

Comment:
Is it really expert annotated?

Reply:
I think so, from the [NTREX paper](https://aclanthology.org/2022.sumeval-1.4/):

> To produce this data set we sent out the original English WMT19 (Barrault et al., 2019) test set (‘newstest2019‘) to professional human translators.

Reply:
Oh wow. 

Then we need to change the text_creation to human translated (and localized if that is the case).

Reply:
We have the following options:
```
    "found",
    "created",
    "machine-translated",
    "human-translated and localized",
    "machine-translated and verified",
    "machine-translated and localized",

```
I think `human-translated and localized` is the best match, even though I cannot find any information about localization in the NTREX paper.